draft-ietf-tls-tls13.md

```diff
@@ -5485,13 +5506,60 @@ compute all future traffic secrets on that connection.  Systems which want such
 guarantees need to do a fresh handshake and establish a new connection with an
 (EC)DHE exchange.
 
-
 ### External References
 
 The reader should refer to the following references for analysis of the TLS record layer:
 {{BMMT15}} {{BT16}} {{BDFKPPRSZZ16}} {{BBK17}}.
 
 
+## Traffic Analysis
+
+TLS is susceptible to a variety of traffic analysis attacks based on
+observing the length and timing of encrypted packets {{MHJT14}}
+{{HCJC16}}.  This is particularly easy to see when there is a small
+set of possible messages to be distinguished, such as for a video
+server hosting a fixed corpus of content, but still provides usable
+information even in more complicated scenarios.
+
+TLS does not provide any specific defenses against this form of attack
+but does include a padding mechanism for use by applications: The
+plaintext protected by the AEAD function consists of content plus
+variable-length padding, which allows the application to produce
+arbitrary length packets as well as cover traffic.  Because the
+padding is encrypted alongside the actual content, an attacker cannot
+directly determine the length of the padding, but may be able to
+measure it indirectly by the use of timing channels exposed during
+record processing (i.e., seeing how long it takes to process a
+record or trickling in records to see which ones elicit a response
+from the server). In general, it is not known how to remove all of
+these channels because even a constant time padding removal function will
+then feed the content into data-dependent functions.  Note that robust
+fingerprinting avoidance will likely lead to inferior performance
+due to delay in transmitting packets and increased traffic volume.
```

Comment:
I would split this last sentence out into a new paragraph and remove the "Note that".